test/packages-src/bonjour/bonjour.bat

```diff
@@ -4,6 +4,7 @@ ECHO %CL_FLAG_LANGUAGE%
 ECHO %CL_ARG_1%
 
 ECHO packge dir: %CL_PACKAGE_DIR%
+ECHO command name: %CL_COMMAND_NAME%
```

Comment:
This shoule be %CL_FULL_COMMAND_NAME%. It fails the integration tests.

Reply:
🤦 of course, my bad. Sorry for the extra back and forth, the good news is that after a first contribution, I can see the test results and iterate before you waste time 😅 .